Octokit/Models/Response/AccountType.cs

```diff
@@ -20,6 +20,12 @@ public enum AccountType
         /// Bot account
         /// </summary>
         [Parameter(Value = "Bot")]
-        Bot
+        Bot,
+
+	/// <summary>
+	/// Mannequin account
```

Comment:
Thoughts on defining this further in the comment? The other account types are pretty self-explanatory but mannequin isn't.